pages/knowledge/colours-and-contrast/graphical-objects/README.md

```diff
@@ -0,0 +1,50 @@
+---
+navigation_title: "Colour contrast for graphical objects"
+position: 5
+changed: "2019-04-21"
+---
+
+# Colour contrast for graphical objects
+
+**The term “graphical object” typically applies to stand-alone icons and to information graphics. The visual complexity of such objects often requires some differentiation with regards to contrast. The focus lies on the parts of the graphical object required to understand the content.**
+
+## Minimal contrast ratio
+
+Version 2.1 of the [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#non-text-contrast) of `3:1` against adjacent colour(s) for the parts, which are required to understand the meaning or content of a graphical object.
+
+## Example: line chart
+
+The following picture shows a simple line chart with overall insufficient contrast. 
+
+![Line chart with one blue trend line and overall low contrast](_media/line-chart-1-low-contrast.png)
+
+The contrast ratio of the main X and Y axis is `2.8:1`, the one of blue trend line is `2.3:1`. The text labels along the X and Y axis have a contrast ratio of `3.9:1` which can also be considered too low because [text has to meet a higher contrast requirement](/knowledge/colours-and-contrast/text) of at least `4.5:1`.  
```

Comment:
Why is there a hard line-break (double whitespace)?

Please remove or replace with a new paragraph.